src/pages/LogView/LoadingPage/index.tsx

```diff
@@ -54,11 +54,8 @@ const LoadingPage: React.FC<LoadingPageProps> = ({ logType }) => {
         url = getResmokeLogURL(buildID, { raw: true });
         htmlLogURL = getResmokeLogURL(buildID, { html: true });
       }
```

Comment:
Somewhat wondering if `logkeeperMetadata` is still something we need at the moment. I guess we will need it in the future, so am ok with it

Reply:
Its used for the task link